hawc/apps/epiv2/managers.py

```diff
@@ -193,6 +194,148 @@ def complete(self):
             "factors",
         ).prefetch_related("design__countries")
 
+    def complete_df(self) -> pd.DataFrame:
+        mapping = {
+            # study
+            **study_df_mapping("study-", "design__study__"),
+            # design
+            "design-pk": "design__pk",
+            # "design-url": "design__get_absolute_url", #todo make a sql function or remove?
+            "design-summary": "design__summary",
+            "design-study_name": "design__study_name",
+            "design-study_design": sql_display("design__study_design", constants.StudyDesign),
+            "design-source": sql_display("design__source", constants.Source),
+            "design-age_profile": "design-age_profile",
+            "design-age_description": "design__age_description",
+            "design-sex": sql_display("design__sex", constants.Sex),
+            "design-race": "design__race",
+            "design-participant_n": "design__participant_n",
+            "design-years_enrolled": "design__years_enrolled",
+            "design-years_followup": "design__years_followup",
+            "design-countries": "design__countries__name",
+            "design-region": "design__region",
+            "design-criteria": "design__criteria",
+            "design-susceptibility": "design__susceptibility",
+            "design-comments": "design__comments",
+            "design-created": "design__created",
+            "design-last_updated": "design__last_updated",
+            # chemical
+            "chemical-pk": "exposure_level__chemical__pk",
+            "chemical-name": "exposure_level__chemical__name",
+            "chemical-DTSXID": "exposure_level__chemical__dsstox__dtxsid",
+            "chemical-created": "exposure_level__chemical__created",
+            "chemical-last_updated": "exposure_level__chemical__last_updated",
+            # exposure
+            "exposure-pk": "exposure_level__exposure_measurement__pk",
+            "exposure-name": "exposure_level__exposure_measurement__name",
+            "exposure-measurement_type": "exposure_level__exposure_measurement__measurement_type",
+            "exposure-biomonitoring_matrix": sql_display(
+                "exposure_level__exposure_measurement__biomonitoring_matrix",  # todo fix default display "?"
```

Comment:
Fix this TODO? Maybe have an optional `default` argument for `sql_display` that defaults to "?", and then specify it here as whatever you think is more appropriate (empty string maybe).